configs/recognition/slowonly/slowonly_r50_video_4x16x1_256e_kinetics400_rgb.py

```diff
@@ -74,7 +74,7 @@
     dict(type='ToTensor', keys=['imgs'])
 ]
 data = dict(
-    videos_per_gpu=8,
+    videos_per_gpu=24,
```

Comment:
v100?

Reply:
Nope, slowfast_4x16 doesn't consume lots of memory, we can fit 24 samples onto a 1080 Ti.